services/path.py

```diff
@@ -160,7 +160,10 @@ def __init__(
 
         if search_term:
             self._paths = [
-                path for path in self.paths if search_term in path.relative_path
+                path
+                for path in self.paths
+                if search_term in path.relative_path
+                or search_term.lower() in path.relative_path
```

Comment:
guess you can just do `path for path in self.paths if search_term in path.relative_path if search_term.lower() in path.relative_path.lower()`